tests/model_serving/model_server/stop_resume/test_raw_stop_resume_model.py

```diff
@@ -0,0 +1,127 @@
+
+
+@pytest.mark.rawdeployment
+@pytest.mark.parametrize(
+    "unprivileged_model_namespace, ovms_kserve_serving_runtime, ovms_raw_inference_service",
+    [
+        pytest.param(
+            {"name": "kserve-raw-stop-resume"},
+            RunTimeConfigs.ONNX_OPSET13_RUNTIME_CONFIG,
+            {
+                "name": ModelFormat.ONNX,
+                "model-version": ModelVersion.OPSET13,
+                "model-dir": "test-dir",
+                "deployment-mode": KServeDeploymentType.RAW_DEPLOYMENT,
+                "stop": "False",
+            },
+        )
+    ],
+    indirect=True,
+)
+class TestStopRaw:
+    @pytest.mark.smoke
+    def test_raw_onnx_rest_inference(
+        self, unprivileged_model_namespace, ovms_kserve_serving_runtime, ovms_raw_inference_service
+    ):
+        """Verify that kserve Raw ONNX model can be queried using REST"""
+        verify_inference_response(
+            inference_service=ovms_raw_inference_service,
+            inference_config=ONNX_INFERENCE_CONFIG,
+            inference_type=Inference.INFER,
+            protocol=Protocols.HTTPS,
+            use_default_query=True,
+        )
+
+    @pytest.mark.parametrize(
+        "patched_raw_inference_service_stop_annotation_true",
+        [pytest.param({"annotations": {Annotations.KserveIo.FORCE_STOP_RUNTIME: "true"}})],
+        indirect=True,
+    )
+    def test_stop_ann_update_to_true_delete_pod_rollout(
+        self,
+        unprivileged_client,
+        unprivileged_model_namespace,
+        ovms_kserve_serving_runtime,
+        ovms_raw_inference_service,
+        patched_raw_inference_service_stop_annotation_true,
+    ):
+        """Verify pod rollout is deleted when the stop annotation updated to true"""
+        """Verify pods do not exist"""
+        verify_no_inference_pods(
+            client=unprivileged_client,
+            isvc=patched_raw_inference_service_stop_annotation_true,
+        )
+
+
+@pytest.mark.rawdeployment
+@pytest.mark.parametrize(
+    "unprivileged_model_namespace, ovms_kserve_serving_runtime, ovms_raw_inference_service",
+    [
+        pytest.param(
+            {"name": "kserve-raw-stop-resume"},
+            RunTimeConfigs.ONNX_OPSET13_RUNTIME_CONFIG,
+            {
+                "name": ModelFormat.ONNX,
+                "model-version": ModelVersion.OPSET13,
+                "model-dir": "test-dir",
+                "deployment-mode": KServeDeploymentType.RAW_DEPLOYMENT,
+                "stop": "True",
+            },
+        )
+    ],
+    indirect=True,
+)
+class TestStoppedResumeRaw:
+    @pytest.mark.smoke
+    def test_stop_ann_true_no_pod_rollout(
+        self,
+        unprivileged_client,
+        unprivileged_model_namespace,
+        ovms_kserve_serving_runtime,
+        ovms_raw_inference_service,
+    ):
+        """Verify no pod rollout when the stop annotation is true"""
+        """Verify pods do not exist"""
+        verify_no_inference_pods(
```

Comment:
The `create_isvc` func no longer would wait for pods when the stop annotation is set, thus it would run quite quickly. In consequence, here `verify_no_inference_pods` may run fast.

I think the likelihood of false positives is significant, given a cluster that is target for running automated tests would be busy. It may take some seconds until the workload is created.

To minimize the likelihood of a false positive, IMO we would need to implement something similar to the `Consistently` func of Ginkgo.